spec/controllers/idv/personal_key_controller_spec.rb

```diff
@@ -42,91 +43,190 @@ def assert_personal_key_generated_for_profiles(*profile_pii_pairs)
 
   let(:idv_session) { subject.idv_session }
 
+  let(:threatmetrix_review_status) { nil }
+
   before do
     stub_analytics
     stub_attempts_tracker
     stub_verify_steps_one_and_two(user, applicant: applicant)
```

Comment:
In #9779 we start applying the FlowPolicyHelper methods to the controller specs. One key pattern is that we are getting rid of the calls to stub_verify_steps_one_and_two, since that doesn't fully set all the idv_session fields and it mocks out the idv_session and user_session. Preference going forward would be to call
```
  before do
    stub_sign_in(user)
    stub_up_to(:verify_info, idv_session: subject.idv_session)
  end
```
then, throughout the spec the session is referenced via `subject.idv_session`. 

See the referenced PR for examples :-)

Reply:
👍 I found I needed to be able to pass `applicant` into stub_up_to  so I tweaked it to allow that